parts/kuberneteskubelet.service

```diff
@@ -32,7 +32,7 @@ ExecStart=/usr/bin/docker run \
         --allow-privileged=true \
         --enable-server \
         --enable-debugging-handlers \
-        --config=/etc/kubernetes/manifests \
```

Comment:
I would have expected this change to result in updating templates.go file.
- Can you run "make build" to make sure that a new templates.go file is generated and add it to your PR?
- Also run "make ci" to run tests locally.